addon/components/frost-radio-button.js

```diff
@@ -66,30 +88,34 @@ export default Component.extend(PropTypeMixin, {
 
   init () {
     this._super(...arguments)
-    Ember.assert(
+    this._setupAssertions()
+  },
+
+  _setupAssertions () {
+    assert(
       `${this.toString()} must be initialized in the yield block of 'frost-radio-group'`,
       /frost-radio-group/.test(this.parentView.toString()))
-    Ember.assert(
+    assert(
       `${this.toString()} must be initialized with a 'value' property`,
-      this.get('value')
+      get(this, 'value')
     )
   },
 
   keyPress (e) {
     if (e.keyCode === 13 || e.keyCode === 32) {
-      if (this.get('disabled') || this.get('groupValue') === this.get('value')) {
+      if (get(this, 'disabled') || get(this, 'groupValue') === get(this, 'value')) {
         return
       }
-      let change = this.get('onChange')
-      if (change && typeof change === 'function') {
-        change(this._createEvent(e, Ember.$(e.target).find('input')[0]))
+      let change = get(this, 'onChange')
+      if (change && typeOf(change === 'function')) {
+        change(this._createEvent(e, $(e.target).find('input')[0]))
       }
     }
   },
 
   change (event) {
-    const onChange = this.get('onChange')
-    if (onChange && typeof onChange === 'function') {
+    const onChange = get(this, 'onChange')
+    if (onChange && typeOf(onChange === 'function')) {
       onChange(this._createEvent(event, event.target))
```

Comment:
I took a look at this and in this case there is a difference so I am going to keep this as is.